spec/controllers/category_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe CategoryController, type: :controller do
+  describe "GET #index" do
+    it "returns http success" do
```

Comment:
the description doesn't match the expectation here. If it's a 301 maybe we can assert that it redirects to a particular page.

Reply:
I've edited this so that it tests for both a 301, and that the redirect ends up on the registers collection page.